pyfr/solvers/acnavstokes/elements.py

```diff
@@ -16,9 +16,9 @@ def set_backend(self, *args, **kwargs):
             return
 
         # Register our flux kernels
+        kernel, kerns = self._be.kernel, self.kernels
```

Comment:
Any chance of better names here?

Reply:
Elsewhere, you use kernels rather than kerns, so let's use that.